packages/component-library/src/_Themes/DataVisualizationTheme.js

```diff
@@ -0,0 +1,335 @@
+import { assign } from "lodash";
+
+// *          *
+// * BRANDING *
+// *          *
+
+// *
+// * Colors
+// *
+
+const civicPrimary = "#1f1123";
+const civicSecondary = "#eb4d5f";
+const civicTertiary = "#716470";
+const civicSecondaryLighter = "#aaa4ab";
+const civicSecondaryLightest = "#f3f2f3";
+
+// *                                  *
+// * DATA VISUALIZATION ONLY BELOW    *
+// *                                  *
+
+const civicCategoricalColor1 = "#DC4556";
+const civicCategoricalColor2 = "#19B7AA";
+const civicCategoricalColor3 = "#1E62BD";
+const civicCategoricalColor4 = "#721D7C";
+const civicCategoricalColor5 = "#FFB226";
+
+const colors = [
+  civicCategoricalColor1,
+  civicCategoricalColor2,
+  civicCategoricalColor3,
+  civicCategoricalColor4,
+  civicCategoricalColor5,
+  civicTertiary
+];
+
+// *
+// * Typography
+// *
+const sansSerif = "'Roboto Condensed', 'Helvetica Neue', Helvetica, sans-serif";
+const letterSpacing = "normal";
+const fontSize = 12;
+const fontWeight = "normal";
+// *
+// * Layout
+// *
+const padding = 8;
+const horizontalBarPadding = 2;
+const baseProps = {
+  width: 650,
+  height: 350,
+  padding: 50
+};
+const tooltipProps = {
+  x: 325,
+  y: 0,
+  orientation: "bottom",
+  pointerLength: 0,
+  cornerRadius: 0
+};
+// *
+// * Labels
+// *
+const baseLabelStyles = {
+  fontFamily: sansSerif,
+  fontSize,
+  fontWeight,
+  letterSpacing,
+  padding,
+  fill: civicPrimary
+};
+
+const centeredLabelStyles = assign({ textAnchor: "middle" }, baseLabelStyles);
+
+const pieLabelStyles = {
+  fontFamily: sansSerif,
+  fontSize: "16px",
+  fontWeight: "bold"
+};
+
+const axisLabelStyles = {
+  fontFamily: sansSerif,
+  fontSize: "14px",
+  fontWeight: "bold"
+};
+
+// *
+// * Strokes
+// *
+const strokeDasharray = "10, 5";
+const strokeLinecap = "round";
+const strokeLinejoin = "round";
+
```

Comment:
Maybe the Victory theme here should be nested under `victory`, since there may be other data visualization libraries in use.

Let say for example we add a Sankey from another library, e.g. `d3-sankey`, that takes a theme in a different format.

Another consideration -- we have a our own bespoke components, one example being GradientScale. What part of the theme object is being used there?

Maybe it looks like this?
```js
{
  victory: { /* victory theme object */ },
  d3-sankey: { /* d3-sankey theme object*/ },
  civic: { /* our own style used for our custom components */ }
}
```

I still like having all of the data visualization styles in one place, defined by constants that are outside of the object or objects that get exported. But also, 🤷‍♂️.

Reply:
Another consideration: should this be DataVisualizationTheme or separated out into ChartTheme and MapTheme?

I think I like keeping it together, but could go either way.

Reply:
Great points here. I'm leaning towards separating out themes that are formatted for different 3rd party sources (i.e., victory visualizations, map visualizations, etc...) and keeping the civic styles as the SOT for any colors / fonts and such that the others need to use.